test/e2e/support.js

```diff
@@ -177,3 +182,31 @@ Cypress.Commands.add('createVault', (params) => {
     });
   });
 });
+
+Cypress.Commands.add('getISTBalance', ({ walletAddress }) => {
```

Comment:
I was thinking to have a generic command "getBalance" which takes denom and walletAddress in input params. Will be re-usable as well.
But that can be a need-nased followup

Reply:
We can. I plan to do a similar thing in `dapp-inter`. 
Over here in `wallet-app`, we are mainly doing IST transactions so didn't go with the reusability part.